autoscheduler/scraper/serializers.py

```diff
@@ -0,0 +1,33 @@
+from rest_framework import serializers
+from .models.course import Course
+from .models.section import Section, Meeting
+
+class CourseSerializer(serializers.ModelSerializer):
+    """ Serializes a course """
+    class Meta:
+        model = Course
+        fields = ['title', 'credit_hours']
+
+class SectionSerializer(serializers.ModelSerializer):
+    """ Serializes a section """
```

Comment:
Maybe more thoroughly explain what a Serializer does in the docstrings of them? Kind of the same thing with the `SerializerMethodField`, as if everyone was completely familiar with Django we wouldn't need to explain this stuff, but I think it is beneficial to explain them just for clarity.